lib/project_types/theme/forms/create.rb

```diff
@@ -0,0 +1,22 @@
+module Theme
+  module Forms
+    class Create < ShopifyCli::Form
+      attr_accessor :name
+      flag_arguments :title, :password, :store
+
+      def ask
+        self.store ||= CLI::UI::Prompt.ask(ctx.message('theme.forms.create.ask_store'), allow_empty: false)
+        ctx.puts(ctx.message('theme.forms.create.private_app', self.store))
+        self.password ||= CLI::UI::Prompt.ask(ctx.message('theme.forms.create.ask_password'), allow_empty: false)
+        self.title ||= CLI::UI::Prompt.ask(ctx.message('theme.forms.create.ask_title'), allow_empty: false)
+        self.name = self.title.downcase.split(" ").join("_")
+
+        errors = []
+        errors << "store" if store.strip.empty?
+        errors << "password" if password.strip.empty?
+        errors << "title" if title.strip.empty?
+        ctx.abort((errors.join(", ") + " cannot be empty").capitalize) unless errors.empty?
```

Comment:
We should ideally build these errors from message keys so they're translatable in the future.